server/controller/User.controller.js

```diff
@@ -0,0 +1,97 @@
+const User = require('../models/Users.model');
+
+const getAllUser = async (req, res) => {
+  try {
+    const users = await User.find({});
+    return res.status(200).json({ users, sucess: true });
+  } catch (error) {
+    console.log(error);
+    return res.status(500).json(error);
+  }
+};
+
+const createUser = async (req, res) => {
+  try {
+    const {
+      name, email, age, password, cpassword,
+    } = req.body;
+    if (!name) {
+      return res.status(400).json({ error: 'Name is required' });
+    }
+    if (!email) {
+      return res.status(400).json({ error: 'Email is required' });
+    }
+    // if(validateEmail(email)){
+    //     return res.status(400).json({error:"Valid Email is required"})
+    // }
+    if (!age) {
+      return res.status(400).json({ error: 'Age is required' });
+    }
+    if (!password) {
+      return res.status(400).json({ error: 'Password is required' });
+    }
+    if (password !== cpassword) {
+      return res.status(400).json({ error: 'Password do not match' });
+    }
+
+    const user = await User.create({
+      name, email, age, password,
+    });
+
+    if (!user) {
+      return res.status(500).json({ message: 'Failed to create user', success: false });
+    }
+
+    return res.status(200).json({ message: 'User successfully created', success: true });
```

Comment:
Use 201 for create status as it is standard. 200 stands for success, but we should follow 201 as it stands for created.